library/std/src/fs.rs

```diff
@@ -1365,6 +1365,30 @@ impl FileTimes {
 impl Permissions {
     /// Returns `true` if these permissions describe a readonly (unwritable) file.
     ///
+    /// Warning: This function does not take ACLs or Unix group membership into
```

Comment:
The doc doesn't have any context about "ACLs", writing it like below would be helpful:

```suggestion
    /// Warning: This function does not take access control lists (ACLs) or Unix group membership into
```